### ISSUE TITLE
make: pep8: Command not found

### ISSUE THREAD

**Issue by ndias** [getting-things-gnome/liblarch#21]
When running make I get the error "make: pep8: Command not found"
The google says pep8 is o longer maintain, the new tool is called pycodestyle
From https://pypi.org/project/pep8/
"Prominently note via warning message that the tool is no longer released as pep8 and will only be fixed in the pycodestyle package"

After changing the pep8 to pycodestyle I get this errors

pyflakes examples liblarch liblarch_gtk tests \
main.py run-tests setup.py
pycodestyle --statistics --count examples liblarch liblarch_gtk tests \
main.py run-tests setup.py
examples/contact_list/contact_list.py:397:1: E305 expected 2 blank lines after class or function definition, found 1
liblarch/filteredtree.py:180:80: E501 line too long (91 > 79 characters)
liblarch/tree.py:196:80: E501 line too long (80 > 79 characters)
liblarch_gtk/__init__.py:20:10: E261 at least two spaces before inline comment
liblarch_gtk/__init__.py:21:33: E261 at least two spaces before inline comment
liblarch_gtk/__init__.py:47:80: E501 line too long (84 > 79 characters)
liblarch_gtk/treemodel.py:62:80: E501 line too long (80 > 79 characters)
liblarch_gtk/treemodel.py:80:80: E501 line too long (82 > 79 characters)
tests/test_liblarch.py:25:1: E402 module level import not at top of file
tests/test_liblarch.py:26:1: E402 module level import not at top of file
tests/test_liblarch.py:27:1: E402 module level import not at top of file
tests/test_liblarch.py:28:1: E402 module level import not at top of file
tests/test_liblarch.py:29:1: E402 module level import not at top of file
tests/test_liblarch.py:30:1: E402 module level import not at top of file
tests/test_liblarch.py:32:1: E402 module level import not at top of file
tests/test_liblarch.py:33:1: E402 module level import not at top of file
tests/test_liblarch.py:34:1: E402 module level import not at top of file
tests/test_liblarch.py:35:1: E402 module level import not at top of file
tests/test_liblarch.py:36:1: E402 module level import not at top of file
tests/test_liblarch.py:37:1: E402 module level import not at top of file
tests/test_signal_testing.py:39:80: E501 line too long (95 > 79 characters)
tests/watchdog.py:51:9: E722 do not use bare 'except'
2       E261 at least two spaces before inline comment
1       E305 expected 2 blank lines after class or function definition, found 1
12      E402 module level import not at top of file
6       E501 line too long (91 > 79 characters)
1       E722 do not use bare 'except'
22
make: *** [Makefile:42: pep8] Error 1

**Comment by nekohayo** [comment]
Thanks for the report, I have a fix in the works now...

**Comment by nekohayo** [comment]
Fixed for me by commit 5eaef0bd and commit 8c90053

Result:

```
 make check && make pep8
./run-tests
pyflakes examples liblarch liblarch_gtk tests \
main.py run-tests setup.py
....................................................................
----------------------------------------------------------------------
Ran 68 tests in 9.112s

OK
pycodestyle --statistics --count --repeat --max-line-length=110 --ignore=E128,W504 \
examples liblarch liblarch_gtk tests main.py run-tests setup.py
tests/watchdog.py:51:9: E722 do not use bare 'except'
1       E722 do not use bare 'except'
1
make: *** [Makefile:42: pep8] Error 1
```

That E722 error I don't care enough to tackle it, but if you feel like it feel free to propose a patch :)
